src/modelgauge/main.py

```diff
@@ -1,40 +1,33 @@
-import click
 import datetime
 import os
 import pathlib
 import warnings
 from typing import List, Optional
 
+import click
+
 from modelgauge.annotator import CompletionAnnotator
 from modelgauge.annotator_registry import ANNOTATORS
 from modelgauge.base_test import PromptResponseTest
 from modelgauge.command_line import (
```

Comment:
I kind of like it when constants are grouped together at the beginning of an import. This is just a preference though,

Reply:
The changes here are done automatically by the standard rules in the Black linter. I can tweak them if you'd like.

Reply:
Hm, I also use black but it doesn't format stuff this way for me. Maybe we have different configurations?